po/es.po

```diff
@@ -8992,7 +8992,7 @@ msgstr "ajustes básicos"
 
 #: ../src/iop/basicadj.c:140
 msgid "apply usual image adjustments"
-msgstr "Emplear los ajustes de imagen habituales"
+msgstr "Emplea los ajustes de imagen más frecuentes"
```

Comment:
@EdgarLux to make translations uniform, it should be "Aplica". Do you agree?

Thanks!